broker-core/src/main/resources/io/zeebe/broker/exporter/debug/index.html

```diff
@@ -0,0 +1,185 @@
+					<div class="input-group-append">
+						<button class="btn btn-outline-secondary my-2 my-sm-0" type="button" onclick="$('#filter-search').val('');fetchRecords();">Clear</button>
+						<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+					</div>
+				</div>
+      </form>
+    </div>
+  </div>
+    </nav>
+    <div class="row">
+    <div id="content" class="col"></div>
+    </div>
+  </div>
+<script id="record-template" type="x-tmpl-mustache">
+  <table class="table table-hover">
+  <thead>
+    <tr>
+      <th scope="col">Record</th>
+      <th scope="col">Key</th>
+      <th scope="col">BPMN Process Id</th>
+      <th scope="col">Element Id</th>
+      <th scope="col">Workflow Instance Key</th>
+      <th scope="col">Scope Instance Key</th>
+      <th scope="col">Element Instance Key</th>
+    </tr>
+  </thead>
+  <tbody>
+  {{ #. }}
+    <tr data-toggle="collapse" data-target="#collapse-{{_id}}" aria-expanded="false" aria-controls="collapse-{{_id}}">
+      <th scope="row" style="text-transform: capitalize;">{{_name}}</th>
+      <td>{{key}}</td>
+      <td>{{_bpmnProcessId}}</td>
+      <td>{{_elementId}}</td>
+      <td>{{_workflowInstanceKey}}</td>
+      <td>{{_scopeInstanceKey}}</td>
+      <td>{{_elementInstanceKey}}</td>
+    </tr>
+    <tr class="collapse" id="collapse-{{_id}}">
+      <td colspan="7"><pre>{{{_json}}}</pre></td>
+    </tr>
+  {{ /. }}
+  </tbody>
+</table>
+</script>
+
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/3.0.0/mustache.min.js" integrity="sha256-1B6REXjRTNQ4IJEUSVYbAqEDRIYP8uKNUo+QgYZUzJM=" crossorigin="anonymous"></script>
+<script>
+  var recordTemplate = $('#record-template').html();
+  Mustache.parse(recordTemplate);
+
+  $("#form-search").submit(function(e) {
+    fetchRecords();
+    e.preventDefault();
+  });
+
+  function fetchRecords() {
+    fetch('/records')
+      .then(function(response) {
+        return response.json()
+      })
+      .then(filterRecords)
+      .then(preProcessRecords)
+      .then(searchRecords)
+      .then(displayRecords);
+  }
+
+  function filterRecords(records) {
+    return records.filter(r => checkboxForRecord(r).is(":checked"));
+  }
+
+  function checkboxForRecord(record) {
+    switch (record.metadata.valueType) {
+      case 'WORKFLOW_INSTANCE':
+        return $('#filter-workflow-instance');
+      case 'JOB':
+        return $('#filter-job');
+      case 'INCIDENT':
+        return $('#filter-incident');
+      case 'TIMER':
+        return $('#filter-timer');
+      case 'MESSAGE_SUBSCRIPTION':
```

Comment:
The value type `MESSAGE` should be also displayed if `Message` is selected.

Reply:
Correct I forgot it, will change